content/applications/services/helpdesk/advanced/track_and_bill.rst

```diff
@@ -0,0 +1,245 @@
+
+The invoice can then be sent to the customer for payment.
+
+Create helpdesk ticket for prepaid services
+-------------------------------------------
+
+To create a *Helpdesk* ticket for prepaid services, navigate to :menuselection:`Helpdesk` and select
+a :guilabel:`Team`. This will reveal that team's :guilabel:`Pipeline`. Click :guilabel:`New` to
+create a new ticket.
+
+On the blank ticket form, create a ticket :guilabel:`Title`, and enter the :guilabel:`Customer`
+information.
+
+When the customer name is added, the :guilabel:`Sales Order Item` field will automatically populate
+with the most recent prepaid sales order item that has time remaining.
+
+Track hours on helpdesk ticket
+------------------------------
+
+Time spent working on a *Helpdesk* ticket is tracked on the *Timesheets* tab on the specific ticket.
+
+On the ticket detail form, click on the :guilabel:`Timesheets` tab and click :guilabel:`Add a line`.
+Choose an :guilabel:`Employee`, add a :guilabel:`Description` of the task, and enter the number of
+:guilabel:`Hours Spent`.
+
+As new lines are added to :guilabel:`Timesheets` tab, the :guilabel:`Remaining Hours on SO` field is
+automatically updated.
+
+.. image:: track_and_bill/track-bill-remaining-hours-total.png
+   :align: center
+   :alt: View of the timesheets tab on a ticket with an emphasis on the remaining hours on an SO.
+
+.. note::
+   If the number of hours on the :guilabel:`Timesheets` tab exceeds the number of hours sold, the
+   :guilabel:`Remaining Hours of SO` will turn red.
+
+As hours are added to the :guilabel:`Timesheets` tab, they are automatically updated in the
+:guilabel:`Delivered` field on the :abbr:`SO (sales order)`, as well.
+
+Invoice post-paid support services
+==================================
+
+When support services are billed based on the amount of time spent on an issue, an invoice cannot be
+created before the total number of hours required to solve the problem have been entered on a
+timesheet. In this case, a service product with the invoicing policy set to :guilabel:`Based on
+Timesheets` (like the one created above) would be used.
+
+Create a sales order with a time-tracked product
+------------------------------------------------
+
+To invoice a customer for post-paid support services, first create a sales order (SO) with the
+*support services product*. To do this, go to :menuselection:`Sales --> Orders --> Quotations -->
+New`.
+
+Fill out the quotation with the customer information.
+
+On the :guilabel:`Order Lines` tab, click :guilabel:`Add a Product`. Select the post-paid services
+product configured in the steps above. After updating any other necessary information,
+:guilabel:`Confirm` the quotation.
+
+.. note::
+   Unlike with the prepaid services quotation, Odoo will not allow an invoice to be created at this
+   time. That is because no services have been performed. In other words nothing has been delivered.
+   Therefore, there is nothing to invoice.
```

Comment:
```suggestion
   Unlike with the prepaid services quotation, Odoo will not allow an invoice to be created at this
   time. This is because no services have been performed; in other words, nothing has been delivered,
   therefore, there is nothing to invoice.
```